pkgs/tools/virtualization/alpine-make-rootfs/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper
```

Comment:
Is the GCC compiler a dependency somehow?

If not, use `stdenvNoCC` instead.

Reply:
Not a dependency, using `stdenvNoCC` now. Thanks for the review!